functionalTest/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "webdriverio-test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "allure generate ./allure-results --clean && allure open"
```

Comment:
Do we not need allure as a dev dependency in this file ?

Reply:
Actually, what is allure? I'd have expected npm test (or npm something) to actually run the test suite. It seems that npm test here fires up a test reports page but doesn't run any tests (unless I'm doing something wrong).
To run the tests I gather the commandline is 'npx wdio wdio.conf.js' - it would be better if that was the action triggered by 'npm test' - the running of the tests is the important part.

Reply:
You may need to add the appropriate dev dependencies to get 'npm test' to run the wdio tests, but that's good. This module should be entirely self contained and require no global dependencies to run - someone (or a CI script) should be able to just run 'npm install; npm test' and it should just work

Reply:
Removed allure related command from the test script and added 'wdio wdio.conf.js' so whenever we use 'npm test' command it will start execution.